server/channels/app/import.go

```
@@ -385,3 +385,15 @@ func (a *App) ListImports() ([]string, *model.AppError) {
 
 	return results, nil
 }
+
+func (a *App) DeleteImport(name string) *model.AppError {
+	filePath := filepath.Join(*a.Config().ImportSettings.Directory, name)
+
+	if ok, err := a.FileExists(filePath); err != nil {
+		return err
+	} else if !ok {
+		return nil
```

Should we error here? If a dev inputs a wrong import name, would they expect an error?

---

@hanzei I have copied the same behaviour that is there in export command. If user inputs a wrong name, we are returning successful message in that case as well. 

I think we should return an error if file doesn't exist. Let me know what you think.

---

Agreed. Can you also please update the export command?

---

added

---

@AulakhHarsh Maybe I missed something: Did you update the logic for the export command and for this command?

---

@hanzei there was input from @isacikgoz that let's not change the idempotency behaviour of the commands.

---

If that is the feecback from @isacikgoz , I don't want to delay the PR even further.